woocommerce/woocommerce-ios
# [HACK week] Deposits summary view

Merge after: #10821 



## Description


In this HACK week project, we're adding the Deposit Overview, so merchants can see their WooPayments balance, and the money currently on its way to their account.

This PR adds the view for the deposits. It's feature flagged until it can be polished for production, but this is planned in a future project. Merging it now will help us avoid issues when we come to finish it off.

Apologies for the length – I could split out the `TopTabView` if that would be helpful?

## Testing instructions


Unit tests will follow in a future PR

Using a store with WooPayments, ideally multi-currency enabled, and some test transactions:
1. Navigate to `Menu > Payments > WooPayments Balances`
2. Observe that the relevant amounts match those shown on the web for your store at `wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Foverview`

Note that only the most recent deposit is shown in the app; the web shows a few recent deposits.

Using a non-WooPayments store
1. Navigate to `Menu > Payments`
2. Observe that the WooPayments Balances item is not shown

Using a WooPayments store again, set up Xcode to build a `Release` scheme to a simulator
1. Navigate to `Menu > Payments`
2. Observe that the WooPayments Balances item is not shown

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/ebf0e9c3-5e1f-493f-9094-d78c1542cdda



---
